doc/migrations/3.3.0_3.3.0plus.md

```diff
@@ -47,8 +47,51 @@ You'll need to iterate over the whole `mam_muc_message` table with the following
 
 ### Cassandra
 
-TODO
+#### Step 1
+
+Please execute the following CQL statements on your MIM database:
+
+```
+USE mongooseim;
+ALTER TABLE mam_muc_message ADD from_jid varchar;
+CREATE INDEX ON mam_muc_message (from_jid);
+DESC mam_muc_message;
+```
+
+#### Step 2
+
+Now you have a schema that is compatible with MIM 3.3.0++ but isn't GDPR-compliant yet because the new column has no meaningful data.
+
+Please pick your favourite scripting/programming language and populate the new column with the help of a [dedicated script](jid-from-mam-muc-script.md).
+You'll need to iterate over the whole `mam_muc_message` table with the following algorithm:
+
+1. extract the whole `mam_muc_message` table:
+```
+SELECT * FROM mam_muc_message;
+```
+1. To make data extraction faster, MongooseIM stores 2 copies of the message in the table:
+```
+cqlsh:mongooseim> SELECT * FROM mam_muc_message WHERE id = 399582233150625537 ALLOW FILTERING;
 
+ room_jid                      | with_nick | id                 | from_jid | message                        | nick_name
+-------------------------------+-----------+--------------------+----------+--------------------------------+-----------
+ room-ad1d999b9e@muc.localhost |           | 399582233150625537 |     null | 0x8350000001...998de2fa8426837 |       Sid
+ room-ad1d999b9e@muc.localhost |       Sid | 399582233150625537 |     null | 0x8350000001...998de2fa8426837 |       Sid
+```
+2. The copy with an empty `with_nick` column must be updated.
+4. Extract the **sender's JID** from the `message` column in the same way as described in the RDBMS migration section. By default cassandra backend uses the `eterm` format.
+3. Update the `from_jid` column with the value of the extracted **sender's JID**:
+```
+cqlsh:mongooseim> UPDATE mam_muc_message SET from_jid = 'username@server' WHERE id = 399582233150625537  AND with_nick = '' AND room_jid = 'room-ad1d999b9e@muc.localhost';
+cqlsh:mongooseim> SELECT * FROM mam_muc_message WHERE id = 399582233150625537 ALLOW FILTERING;
+
+ room_jid                      | with_nick | id                 | from_jid        | message                        | nick_name
+-------------------------------+-----------+--------------------+-----------------+--------------------------------+-----------
+ room-ad1d999b9e@muc.localhost |           | 399582233150625537 | username@server | 0x8350000001...998de2fa8426837 |       Sid
+ room-ad1d999b9e@muc.localhost |       Sid | 399582233150625537 |            null | 0x8350000001...998de2fa8426837 |       Sid
+
+```
+3. 
```

Comment:
Leftover?

Reply:
nope, added intentionally to show which record is updated.

Reply:
ooops, i haven't noticed that `3.` :)